cmd/common/trace/exec.go

```diff
@@ -16,104 +16,52 @@ package trace
 
 import (
 	"fmt"
-	"strings"
 
 	"github.com/spf13/cobra"
 
 	commonutils "github.com/kinvolk/inspektor-gadget/cmd/common/utils"
-	"github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/exec/types"
+	execTypes "github.com/kinvolk/inspektor-gadget/pkg/gadgets/trace/exec/types"
 )
 
-type ExecParser struct {
-	commonutils.BaseParser[types.Event]
-}
-
-func newExecParser(outputConfig *commonutils.OutputConfig, prependColumns []string) TraceParser[types.Event] {
-	columnsWidth := map[string]int{
-		// TODO: Move Kubernetes metadata columns to common/utils.
-		"node":      -16,
-		"namespace": -16,
-		"pod":       -30,
-		"container": -16,
-		"pid":       -7,
-		"ppid":      -7,
-		"pcomm":     -16,
-		"ret":       -4,
-		"args":      -24,
+func newExecParser(outputConfig *commonutils.OutputConfig, metadataTag string) (TraceParser[execTypes.Event], error) {
```

Comment:
Maybe you can received `opts...` here and use `return newExecParser(outputConfig, WithMetadata(commonutils.KubernetesTag))` in the caller?

Reply:
I considered it, but I thought that it would have been strange to use options from another library here. However, let's say that [`Option`](https://github.com/kinvolk/inspektor-gadget/blob/841b0d8f511bc879385e1ae13102f409ca2d3fd8/cmd/common/utils/parser-tableformatter.go#L34) is the generic option for all the parsers, and it will indeed make the code clear. In addition, I really don't like passing `""` as an argument, so I will do it! 😝